contrib/udfs/pom.xml

```diff
@@ -30,6 +31,12 @@
   <artifactId>drill-udfs</artifactId>
   <name>contrib/drill-udfs</name>
 
+  <repositories>
+    <repository>
+      <id>Jabylon Repository</id>
+      <url>http://www.jabylon.org/maven/</url>
```

Comment:
I don't think that we should add one more repository. Could you please use dependencies from maven central?